python/smashbox/utilities/__init__.py

```diff
@@ -270,8 +270,10 @@ def run_ocsync(local_folder, remote_folder="", n=None, user_num=None):
     for i in range(n):
         t0 = datetime.datetime.now()
         cmd = config.oc_sync_cmd+' '+local_folder+' '+oc_webdav_url('owncloud',remote_folder,user_num) + " >> "+config.rundir+"/%s-ocsync.step%02d.cnt%03d.log 2>&1"%(reflection.getProcessName(),current_step,ocsync_cnt[current_step])
-        runcmd(cmd, ignore_exitcode=True)  # exitcode of ocsync is not reliable
         logger.info('sync cmd is: %s',cmd)
+        exitcode,stdout,stderr = runcmd(cmd, ignore_exitcode=True)  # exitcode of ocsync is not reliable
+        if exitcode != 0:
+            logger.warning('sync cmd failed(?) with exitcode: %d, stdout: %s, stderr: %s', exitcode, stdout, stderr)
```

Comment:
stdout and stderr are empty because they are written to a file


````

2015-06-25 11:10:23,473 - WARNING - sharer - Non-zero exit code 1 from command '/var/lib/jenkins/jobs/smashbox/workspace/TEST_NAME/test_shareGroup/compiler/g++/label/master/mirallBranch/v1.8.1/qt/qt4/serverVersion/owncloud8.1/client/owncloudcmd --trust /var/lib/jenkins/build/smashdir/test_shareGroup/sharer/ ownclouds://test1:test@test1.owncloud.com/owncloud8.1/remote.php/webdav/ >> /var/lib/jenkins/build/smashdir/test_shareGroup/sharer-ocsync.step03.cnt000.log 2>&1'